src/base/configuration.cpp

```diff
@@ -110,8 +110,105 @@ std::string Configuration::getVersionExtra() {
   return CVC4_EXTRAVERSION;
 }
 
+std::string Configuration::copyright() {
+  std::stringstream ss;
+  ss << "Copyright (c) 2009-2017 by the authors and their institutional\n"
+     << "affiliations listed at http://cvc4.cs.stanford.edu/authors\n\n";
+
+  if (Configuration::licenseIsGpl()) {
+    ss << "This build of CVC4 uses GPLed libraries, and is thus covered by\n"
+       << "the GNU General Public License (GPL) version 3.  Versions of CVC4\n"
+       << "are available that are covered by the (modified) BSD license. If\n"
+       << "you want to license CVC4 under this license, please configure CVC4\n"
+       << "with the \"--bsd\" option before building from sources.\n\n";
+  } else {
+    ss << "CVC4 is open-source and is covered by the BSD license (modified)."
+       << "\n\n";
+  }
+
+  ss << "THIS SOFTWARE IS PROVIDED AS-IS, WITHOUT ANY WARRANTIES.\n"
+     << "USE AT YOUR OWN RISK.\n\n";
+ 
+  ss << "CVC4 incorporates code from ANTLR3 (http://www.antlr.org).\n"
+     << "See licenses/antlr3-LICENSE for copyright and licensing information."
+     << "\n\n";
+
+  if (Configuration::isBuiltWithAbc()) {
```

Comment:
We can probably remove this, no? ABC is mentioned again on line 153.

Reply:
Yes, good catch. That's a left over...